manifests/squid.yaml.in

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: spice-proxy
+spec:
+  ports:
+    - port: 3128
+      targetPort: spice-proxy
+  externalIPs:
+    - "{{ master_ip }}"
```

Comment:
afaik we can't use config map values in services in these sections. It is only used to fill externalPIs. If it is somewhere in a pod used, you are right, we can do it with a configMap, but we can't get rid of it right now.